client/src/modules/cohort/index.jsx

```diff
@@ -122,8 +128,26 @@ class Cohort extends PureComponent {
     return cohortDetails && cohortDetails.isAdmin;
   };
 
+  handleArchivedListsVisibility = id => () => {
+    const { areArchivedListVisible } = this.state;
+    this.setState({ areArchivedListVisible: !areArchivedListVisible });
+    this.handleArchivedListsData(id);
+  };
```

Comment:
if we should follow our current approach there should be show and hide methods, but I would leave this toggle method just for that case, because this is toggling a view indeed.

Reply:
You're right and I was wondering about it. But this is a classic toggle case. Here is one button for switching visibility and there is no other place where this method is invoke or where we can change visibility of archived cohorts.
When we are editing list or adding cohort we invoke e.g. 'hide' method in different places at different moments: on cancel button click, on escape button click, on edit method success. Invoking toggling method in those cases would be non-intuitive...